integration/internal/emulator/blockchain.go

```diff
@@ -1001,19 +1001,10 @@ func (b *Blockchain) systemChunkTransaction() (*flowgo.TransactionBody, error) {
 		systemChunkTransactionTemplate,
 		templates.Environment{
 			RandomBeaconHistoryAddress: serviceAddress.Hex(),
+			EVMAddress:                 serviceAddress.Hex(),
 		},
 	)
 
-	// TODO: move this to `templates.Environment` struct
-	script = strings.ReplaceAll(
-		script,
-		`import EVM from "EVM"`,
-		fmt.Sprintf(
-			"import EVM from %s",
-			serviceAddress.HexWithPrefix(),
-		),
-	)
-
 	script = strings.ReplaceAll(
```

Comment:
Likewise here. Not sure if `Migration` is intended to be treated differently or this can be removed.